addons/stock_dropshipping/models/res_company.py

```diff
@@ -62,12 +62,7 @@ def _create_dropship_picking_type(self):
     def create_missing_dropship_picking_type(self):
         company_ids = self.env['res.company'].search([])
         company_has_dropship_picking_type = (
-            self.env['stock.picking.type']
-            .search([
-                ('default_location_src_id.usage', '=', 'supplier'),
-                ('default_location_dest_id.usage', '=', 'customer'),
-            ])
-            .mapped('company_id')
+            self.env['stock.picking.type'].search([("code", "=", "dropship")]).company_id
```

Comment:
You can put it on one line and get rid of the extra parenthesis. :slightly_smiling_face:

Reply:
Hi @clesgow sorry for the late reply. I have checked, and it overpasses 120 cols, that it's your line length limit if I'm not wrong.

Reply:
@clesgow any news? cc @svs-odoo